ext/pcntl/tests/pcntl_rfork.phpt

```diff
@@ -20,5 +26,6 @@ if ($pid > 0) {
 ?>
 --EXPECTF--
 *** Test with no flags ***
+ValueError: pcntl_rfork(): Argument #1 ($flags) RFFDG and RFCFDG are mutually exclusive
```

Comment:
This wording doesn't make sense to me. I don't have a good idea either, but something like `Argument #1 ($flags) cannot include both RFFDG and RFCFDG, because these flags are mutually exclusive`.

Reply:
fair enough. would you agree with 'must not include both...' to be in same style as similar messages?

Reply:
Yes of course!